shared/src/commonMain/kotlin/co/touchlab/kampkit/Koin.kt

```diff
@@ -63,6 +64,16 @@ private val coreModule = module {
     // See https://github.com/touchlab/Kermit
     val baseLogger = Logger(config = StaticConfig(logWriterList = listOf(platformLogWriter())), "KampKit")
     factory { (tag: String?) -> if (tag != null) baseLogger.withTag(tag) else baseLogger }
+
+    single {
+        BreedRepository(
```

Comment:
I think it'd be great for readability to have named arguments, especially since this is a widely referenced sample project.